rustls/src/error.rs

```diff
@@ -107,6 +110,24 @@ pub enum Error {
     Other(OtherError),
 }
 
+/// Specific failure cases from [`crate::crypto::signer::CertifiedKey::keys_match`].
```

Comment:
For the docstring here and on its variants, might be nice to drop the whole path from the references, and hide the link target. (By doing like a `[keys_match]: crate::crypto::signer::CertifiedKey::keys_match` or whatever at the bottom.)

Reply:
Done!